templates/files/pimcore/.php-cs-fixer.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * PHP CS Fixer configuration for this project. For documentation @see https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/config.rst
+ */
+
+$finder = PhpCsFixer\Finder::create()                       // @phpstan-ignore class.notFound (class lives in phar file and can't be seen)
+    ->in(__DIR__)
+    ->exclude([
+        ...require(__DIR__ . '/vendor/youwe/testing-suite/config/pimcore/php-cs-fixer/excludes.php'),
+
+        /**
+         * List your application specific exclude paths here
+         */
+
+        // example: 'autogenerated_content',
```

Comment:
- [ ] Please update the exclusion generator module to make sure you can easily create a list of excludes for old projects.

Reply:
Will discuss with @Anve94 who built that tool initially. Since PHP CS Fixer is actually very good able to resolve all issues it found automatically (I tested on one project, after auto-fix it didn't had any complains at all anymore, so no manual intervention necessary), I'm thinking about skipping exclusions and just have your project fixed straight away.